functions-and-scope.js

```diff
@@ -15,6 +15,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6
+console.log("opdracht 1: ");
+
+let countStudents = null;
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        countStudents = countStudents + 1;
```

Comment:
Je zou dit nog wat kunnen versimpelen
```suggestion
        countStudents++;
```